client/src/components/table/columns.tsx

```diff
@@ -182,18 +191,18 @@ export const googleDocColumn = buildColumn({
   },
 });
 
-export const deadlineColumn = buildColumn({
-  title: 'Pitch Compleltion Deadline',
-  width: 2,
-  extractor: function getDeadline(pitch: BasePopulatedPitch) {
-    return new Date(pitch.deadline).toLocaleDateString();
+export const deadlineColumn = buildColumn<BasePopulatedPitch>({
```

Comment:
All of the edits on these widths aren't messing with the pitch doc right?

Reply:
They shouldn't because these are new columns.